### PR TITLE
HA ping endpoint, rate limit whitelist

### PR DESCRIPTION
**Pull request by zxfrank**
This is the 1st time I play with go, I wanted to add a ping route for haproxy to monitor service and allow some networks to bypass rate limits.

comments welcome

### CODE REVIEW

**Comment by sdwolfe32** [comment]
IP whitelisting can easily be done by splitting on `,` and checking the existence of an IP in a map

**Comment by sdwolfe32** [comment]
Keep in mind that running the server with both environment variables empty will disable the limiting middleware all together

File: config/config.go [hunk 1]
```diff
@@ -12,6 +12,10 @@ var (
 	ServeWeb, _ = strconv.ParseBool(getEnv("SERVE_WEB", "false"))
 	// Port defines the port used by the api server
 	Port = getEnv("PORT", "8080")
+	// NoRestartOnBlacklist defines if dyno restart on blacklisted. use if you have a static ip
+	NoRestartOnBlacklist = getEnv("NO_RESTART", "false")
```
Comment (sdwolfe32) [inline config/config.go]: This shouldn't be needed since the heroku package doesn't talk to heroku if the heroku specific environment variables aren't set

Comment (zxfrank) [reply config/config.go]: yeah that was a "not so clever" way to bypass the "reboot try" and mute the log

**Pull request closed without merging**